app/queries/get_embedded_content.rb

```diff
@@ -0,0 +1,39 @@
+module Queries
+  class GetEmbeddedContent
+    attr_reader(:target_content_id)
+
+    def initialize(target_content_id)
+      self.target_content_id = target_content_id
+    end
+
+    def call
+      if Document.find_by(content_id: target_content_id).nil?
+        message = "Could not find an edition to get embedded content for"
+        raise CommandError.new(code: 404, message:)
+      end
+
+      Presenters::Queries::EmbeddedContentPresenter.new(
+        target_content_id,
+        host_editions,
+      ).present
```

Comment:
Can/should we use the `.present` class method here, or is there a reason we need to use the instance method?

Reply:
I think I followed [this Presenter as a template](https://github.com/alphagov/publishing-api/blob/6cee925ae331e7cc059b1432033df8d1d343a103/app/presenters/queries/link_set_presenter.rb#L12) which had the instance method and then I didn't have a test to drive me to add the `self.present(…)` that calls the initialiser internally. I see now that other presenters are doing that so I'll add that too to keep it consistent 👍